package.json

```diff
@@ -808,5 +809,11 @@
     ],
     "enabledApiProposals": [
         "terminalDataWriteEvent"
-    ]
+    ],
+    "__metadata": {
```

Comment:
What's this for?

Reply:
When I run `npm run test` it adds this. It also changes the spaces to tabs.

Reply:
Yeah, it's caused by the @vscode/test-electron package. Other extensions just don't commit the changes. Sorta annoying...

https://github.com/microsoft/vscode/issues/148975

Reply:
Ugh. Well, try removing that and see if the tests run successfully.